src/main/resources/descriptions/holiday_descriptions.properties

```diff
@@ -63,7 +63,6 @@ holiday.description.EGYPT_23_JULY_REVOLUTION       = July 23rd Revolution Day
 holiday.description.EGYPT_25_JAN_REVOLUTION        = January 25th Revolution Day
 holiday.description.EGYPT_30_JUNE_REVOLUTION       = June 30th Revolution Day
 holiday.description.EGYPT_ARMED_FORCES_DAY         = Armed Forces Day
-holiday.description.EGYPT_COPTIC_CHRISTMAS         = Coptic Christmas Day
```

Comment:
I wonder why this translation key used in the library was deleted? This is the Christmas key for Egypt.

Reply:
It was duplicated resource key. A few lines upper these files have the same line. So it was just cleanup and it was done in a separate commit.

Reply:
Found the duplicate. Good catch.